src/style.css

```diff
@@ -46,10 +46,6 @@ input::placeholder {
 
 .desc {
   margin-left: px;
-
-  /* width: 90%; */
-
-  /* background-color: rebeccapurple; */
```

Comment:
- [ ] Kindly remove any commented code in your CSS file, since it is not good practice to have them.